components/Meeting.tsx

```diff
@@ -34,6 +34,14 @@ export default function Meeting({
   youtube = '',
   bg = ''
 }: MeetingProps) {
+  const [isClient, setIsClient] = useState(false);
+
+  useEffect(() => {
+    setIsClient(true);
+  }, []);
+
+  if (!isClient) return null;
+
```

Comment:
Can you please explain why we need `useEffect` and `isClient` thing over here?

Reply:
We get a hydration error , so we have to turn it into a client side component . This remedy is from the official nextjs docs.